daos/BookmarkDao.ts

```diff
@@ -27,6 +27,7 @@ export default class BookmarkDao implements BookmarkDaoI {
   private constructor() {}
 
   async bookmarkATuit(tid: string, uid: string): Promise<Bookmark> {
```

Comment:
You need a js doc for all the functions

Reply:
Is there a need for you to most of the package-lock.json? Would that affect later install of the npm?

Reply:
There is no need to add js doc in functions that are implementation of an interface.

Reply:
Merging as no changes are needed

Reply:
Post man collection for bookmarks:

get all bookmark of a user: http://localhost:5002/users/638f5daf5561b563b87b7577/bookmarks
Create bookmark: 
http://localhost:5002/users/638f5daf5561b563b87b7577/bookmarks/635c50ea9567e5d3fbe978b5
delete bookmark: http://localhost:5002/users/638f5daf5561b563b87b7577/bookmarks/635c50ea9567e5d3fbe978b5